src/frontend/js/search-chats.js

```diff
@@ -0,0 +1,240 @@
+      color: black;
+      font-weight: 200;
+      max-height: 0px;
+      overflow-y: hidden;
+      transition: .5s;
+      text-align: center;
+    }
+
+    #searchChats input:focus {
+      border-bottom-left-radius: 0px;
+      border-bottom-right-radius: 0px;
+    }
+
+    #searchChats input:focus ~ .dropdown {
+      max-height: 325px;
+      margin-top: 3px;
+      border-bottom-left-radius: 10px;
+      border-bottom-right-radius: 10px;
+      overflow-y: auto;
+    }
+
+    .dropdown::-webkit-scrollbar {
+      background: lightgray;
+      border-radius: 10px;
+      width: 5px;
+    }
+
+    .dropdown::-webkit-scrollbar-thumb {
+      background: gray;
+      width: 1px;
+      height: 1px;
+    }
+
+    .dropdown > div {
+      min-height: 60px;
+      padding: 8px 10px;
+      font-weight: bold;
+      transition: 0.5s;
+      cursor: pointer;
+      display: flex;
+      align-items: center;
+      gap: 1em;
+    }
+
+    .dropdown p {
+      overflow-x: hidden;
+      white-space: nowrap;
+      text-overflow: ellipsis;
+      max-width: calc(100% - 80px);
+    }
+
+ 
+    .dropdown .nofound {
+        color: gray;
+        font-size: 10pt;
+        align-self: center;
+        font-weight: lighter;
+    }
+
+
+    .material-icons {
+      font-family: "Material Icons";
+      font-weight: normal;
+      font-style: normal;
+      font-size: 24px;
+      line-height: 1;
+      letter-spacing: normal;
+      text-transform: none;
+      display: inline-block;
+      white-space: nowrap;
+      word-wrap: normal;
+      direction: ltr;
+      -webkit-font-smoothing: antialiased;
+      cursor: pointer;
+    }
+
+    .dropdown > div:hover {
+      background: lightgray;
+    }
+
+    #searchChats .containerInput {
+      position: relative;
+    }
+
+    .containerInput input:focus ~ span {
+      opacity: 0;
+      visibility: hidden;
+    }
+
+    .containerInput span {
+      position: absolute;
+      transform: translate(-50%, -50%);
+      top: 50%;
+      right: 0px;
+      z-index: 5;
+      color: #6f6f6f;
+      transition: 0.5s;
+    }
+
+    .dropdown .avatar {
+      background: gray;
+    }
+
+    .avatar {
+      width: 50px;
+      height: 50px;
+      border-radius: 50%;
+      
+    }
+
+    .conversazioni:has(#searchChats input:focus) .elencoFarmacie {
+      opacity: 0;
+    }
+
+    `
+
+    constructor() {
+        super()
+        this.elencoFarmacie = []
+        this.query = ''
+    }
+
+    render() {
+        return html`
+        <div id="searchChats">
+            <div class="containerInput">
+
+                <input type="text" placeholder="cerca farmacie" @input=${this.filtra} />
+                <span class="material-icons"> search </span>
+
+                <div class="dropdown">
+                        ${
+                            this.mostraSuggerimenti()
+                        }
+                </div>
+
+            </div>
+        </div>
+        `
+    }
+
+    filtra(event) {
```

Comment:
usiamo l'inglese per i nomi di classi, funzioni o variabili

Reply:
modificato in setFilter()